gtest/src/manager.rs

```diff
@@ -67,7 +67,7 @@ impl Actor {
             let mut prog = maybe_prog
                 .take()
                 .expect("actor storage contains only `Some` values by contract");
-            if let Program::Genuine(p, _) = &mut prog {
+            if let Program::Genuine(p, _, _) = &mut prog {
```

Comment:
We can condense wildcards (https://rust-lang.github.io/rustfmt/?version=master&search=condense_wildcard_suffixes#condense_wildcard_suffixes)
```suggestion
            if let Program::Genuine(p, ..) = &mut prog {
```